client/src/api/profiles.js

```diff
@@ -7,5 +7,5 @@ export const newProfile = (profileData, token) => {
       Authorization: "Bearer " + token
     }
   };
-  return fetch("/api/profile", postProfile).then(res => res.ok);
+  return fetch("/api/profiles", postProfile).then(res => res.ok);
```

Comment:
note that Andrei is changing this, so you should probably wait for his change and merge your change after rebasing on his